credit-risk/src/app.py

```diff
@@ -1,28 +1,234 @@
-from h2o_wave import app, Q, ui, main
+import pandas as pd
+from h2o_wave import app, data, main, Q, ui
 
-from .views.home import render_home
-from .views.customer import render_customer_page, handle_approve_click, handle_reject_click
+from .model import Model
+
+
+TRAIN_CSV = "./data/Kaggle/CreditCard-train.csv"
+TEST_CSV = "./data/Kaggle/CreditCard-train.csv"
+ID_COLUMN = "ID"
+TARGET_COLUMN = "default.payment.next.month"
+# Recommend rejecting customer's with default probability appove this amount 
+APPROVAL_THRESHOLD = 0.35
 
 
 def init(q: Q):
-    q.app.customer_status = {}
+    # build model
+    q.app.model = Model(TRAIN_CSV, ID_COLUMN, TARGET_COLUMN)
+    # prep df for analyzation
+    df = pd.read_csv(TEST_CSV).head(20)
+    df.drop(TARGET_COLUMN, axis=1, inplace=True)
+    # analyze df
+    q.app.predictions_df = q.app.model.predict(df)
+    q.app.contributions_df = q.app.model.contrib(df)
+    # prep df for viewing
+    df['Default Prediction Rate'] = q.app.predictions_df.round(4)
+    df.insert(loc=0, column='Status', value='Pending', allow_duplicates=True) 
+    q.app.customer_df = df
+
+
+def layout(q: Q):
+    q.page['meta'] = ui.meta_card(
+        box='', 
+        title='Credit Risk', 
+        layouts=[
+            ui.layout(
+                breakpoint='xs',
+                zones=[
+                    ui.zone('header'),
+                    ui.zone('customer_table'),
+                    ui.zone(
+                        'customer_page',
+                        zones=[
+                            ui.zone('customer_risk_explanation'),
```

Comment:
When I look at this on a small screen the X and Y axis of the chart swap. Is this intentional or a code / wave bug? 
<img width="570" alt="image" src="https://user-images.githubusercontent.com/4793561/118318209-e76db280-b4ad-11eb-8ca9-f4af7cc24b6e.png">

Reply:
Definitely not intentional. It also doesn't rotate back if the screen is enlarged so I would consider it a bug.

Reply:
This is the same bug I faced: https://github.com/h2oai/wave/issues/801 - should be fixed soon.